src/components/TranslationBanner.tsx

```diff
@@ -40,7 +40,7 @@ const TranslationBanner: React.FC<IProps> = ({
       as="aside"
       display={isOpen ? "block" : "none"}
       bottom={{ base: 0, md: 8 }}
-      right={{ base: 0, md: 8 }}
+      insetInlineEnd={{ base: 0, md: 8 }}
```

Comment:
@wackerow yep, that was to be expected. I only made updates to those that were very noticeable.

I don't expect everything to be directionally aware in design, but I can go ahead and review in this PR the components with props that should be flipped like what you showed.

@pettinarip do advise still your thoughts and if this should be handled here. 😁

Reply:
@wackerow good observation. Probably I wasn't clear enough but the idea is to do the initial setup for rtl support in the nextjs repo by using chakra.

I agree that doing the entire adaptation now would be scope creep. That is why I think we should do all that heavy lifting after we finish the migration, as a separate effort.

Do you see some other issues with doing this in two parts?

Reply:
>  agree that doing the entire adaptation now would be scope creep.

Agree as well... just thinking. Adding `dir="rtl"` to the `html` tag will flip the entire layout... two items next to each other will be flipped, and any `mr/marginRight` etc. being used is unlikely to look correct anymore.

A small example, I went to https://ethereum.org/ar/nft/ and manually added `dir="rtl"` to the html tag in the DOM:

<img width="1326" alt="image" src="https://github.com/ethereum/ethereum-org-fork/assets/54227730/67d3fe5a-e4aa-415c-b2a4-7341396410a3">

The icon in this banner has a "margin right" on it... which only works appropriately when it is to the left of the text... in the image above you see the gap between the icon and the text is gone. 

---

I'm not sure exactly how many of these will surface with this switch, but overall I don't think it will be all that serious in the short term, and think it's probably best to prevent the scope creep, get this PR in without being blocked by updating everything, and splitting that as a near-future task.